libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -738,36 +738,36 @@ class AC_PosControl
     float       _yaw_rate_target_rads;       // desired yaw rate in radians per second calculated by position controller
 
     // position controller internal variables
-    Vector3p    _pos_estimate_neu_m;
-    Vector3p    _pos_desired_neu_m;        // desired location, frame NEU in m relative to the EKF origin. This is equal to the _pos_target minus offsets
-    Vector3p    _pos_target_neu_m;         // target location, frame NEU in m relative to the EKF origin. This is equal to the _pos_desired_neu_m plus offsets
-    Vector3f    _vel_estimate_neu_ms;
-    Vector3f    _vel_desired_neu_ms;       // desired velocity in NEU m/s
-    Vector3f    _vel_target_neu_ms;        // velocity target in NEU m/s calculated by pos_to_rate step
-    Vector3f    _accel_desired_neu_mss;    // desired acceleration in NEU m/s² (feed forward)
-    Vector3f    _accel_target_neu_mss;     // acceleration target in NEU m/s²
+    Vector3p    _pos_estimate_ned_m;
+    Vector3p    _pos_desired_ned_m;        // desired location, frame NED in m relative to the EKF origin. This is equal to the _pos_target minus offsets
```

Comment:
I wonder if the last part of this comment should perhaps mention "_pos_target_ned_m" instead of just "_pos_target"?

Reply:
fixed